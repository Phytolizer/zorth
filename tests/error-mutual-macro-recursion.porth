macro bar
  foo
end

macro foo
  bar
end

foo
