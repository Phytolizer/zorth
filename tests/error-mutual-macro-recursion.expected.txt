:i returncode 1
:i argc 0
:b stdin 0

:b stdout 0

:b stderr 91
tests/error-mutual-macro-recursion.porth:2:3: ERROR: macro exceeded expansion limit (1000)

